### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by Justdigit** [ShadyNawara/WLEDAuraSync#3]
Hi i am using this great feature for a while now.....is it possible to update wled to latest version?? Many thanks

**Comment by Justdigit** [comment]
Is this project abandoned??

**Comment by ShadyNawara** [comment]
not really, I have just been a little bit busy. I will look into updating it as soon as I get a chance. In the meantime, I would recommend [OpenRGB](https://openrgb.org), thats what I use personally now and it can link to other WLED devices or even control them fairly easily.

**Comment by dtk1985** [comment]
> In the meantime, I would recommend [OpenRGB](https://openrgb.org), thats what I use personally now and it can link to other WLED devices or even control them fairly easily.

May I ask how does your setup look? Can you also control OpenRGB lights (for example: neopixel strip connected to MB headers) from WLED, or just the other way around?

**Comment by ShadyNawara** [comment]
I am not syncing the 2 any more. I have this wall light controlled by WLED and my computer is controlled separately with OpenRGB. I think that out of the box, OpenRGB can control WLED but WLED cannot control OpenRGB.

**Comment by Justdigit** [comment]
> I am not syncing the 2 any more. I have this wall light controlled by WLED and my computer is controlled separately with OpenRGB. I think that out of the box, OpenRGB can control WLED but WLED cannot control OpenRGB.

Yes thats correct.... but i "need" wled to control my Motheboard led. Although its just the RAM RGB and the mobo leds itself.
All my RGB fans are controled by a Wled module.

I like to do simracing and i have DIYHue controling my WLED modules. This way i have interactive ambilight in my room.

**Comment by Justdigit** [comment]
If you are interested here you see the effect 
https://www.youtube.com/watch?v=I73VIQvR_qY&t=1619s&ab_channel=BoostedMedia

And here DiyHue. Pretty awesome project
https://diyhue.org/
